api/models/FollowedTag.js

```diff
@@ -0,0 +1,16 @@
+module.exports = bookshelf.Model.extend({
```

Comment:
can we call this TagFollow instead, in order to emphasize that the record represents an act of following, not a tag?

(feeling a similar weirdness about CommunityTag as well, but not sure what else to call it -- maybe TagCommunityMembership, which is long but consistent with what we're already doing e.g. PostProjectMembership)